scania-digital-design-system/sdds
# fix(side-menu): replace <a> with <button>

**Describe pull-request**  
Replaced the link with button for better accessability and correct html

BREAKING CHANGE: Replaced links with button

**Solving issue**  
Fixes: [AB#2706](https://dev.azure.com/scaniadigitaldesignsystem/3d9eb9c6-0045-473d-bc1d-e842dd779200/_workitems/edit/2706)

**How to test**  
1. Go to storybook link below
2. Check in Components -> Side-menu
3. Check that the close/expand buttons still works as intended.

**Screenshots**  
-

**Additional context**  
-
